Makefile

```diff
@@ -310,6 +315,6 @@ LVM_OPERATOR_UNINSTALL ?= true
 # Build and run lvm tests.
 e2e-test:
```

Comment:
```suggestion
e2e-test: ginkgo
```

Reply:
Thanks Nithya for the suggestion,
It is working and is able to install ginkgo if not already installed.
It creates a ginkgo file inside the bin package if we already dont have that.